contribute/creating_package.md

```diff
@@ -55,30 +52,32 @@ The package called "tiddlywinks" can now be created using the template `t` by ru
 t("tiddlywinks")
```

Comment:
Packages in Julia generally have a capital first letter.

Reply:
done